docker/Dockerfile.base.18.04

```diff
@@ -63,6 +74,9 @@ RUN mkdir -p /root/.mujoco && \
   ln -s $HOME/.mujoco/mujoco200_linux $HOME/.mujoco/mujoco200
   ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/root/.mujoco/mujoco200/bin
 
+
+RUN python3.5 -m ensurepip && python3.5 -m pip install --upgrade pip
```

Comment:
for `RUN` statements, please use the convention of one statement per line, terminated with `&& \`